src/insert_sort.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+import timeit
+
+
+best_case = [x for x in range(0, 1000)]
+worst_case = best_case[::-1]
+
+
+def insert_sort(l):
+    """Sort a list by inserting values in order after comparing each value."""
+    if len(l) <= 1:
+        return
+    else:
+        for idx in range(1, len(l)):
```

Comment:
Good insertion sort implementation! Only improvement I would want to see is to make a more meaningful/readable variable name than just "`l`" for the input list.